app/views/pages/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1>Blog Pages</h1>
+
+<% #= @pages.first.title %>
```

Comment:
Usually on the `index page` we wanna show `ALL` data (here means all pages) to the views.
Here we only show the `first` page according to line 3, and it'll always show this page.
For practical/real projects this is not the favored code in the `index` view.
Since the video did not mention/ask us to write a complete index.html.erb, it works fine now for this assignment.

Reply:
I mean you are doing great, everything works as the assignment required.
I admire your hard work and just want you to know: as a developer we want to write our `show actions` for better query/error handling.